OpenDreamRuntime/Procs/DebugAdapter/DreamDebugManager.cs

```diff
@@ -826,6 +832,26 @@ private void HandleRequestDisassemble(DebugAdapterClient client, RequestDisassem
         requestDisassemble.Respond(client, DisassemblySkipTake(output, requestedPoint, requestDisassemble.Arguments.InstructionOffset ?? 0, requestDisassemble.Arguments.InstructionCount));
     }
 
+    private void HandleRequestHotReloadInterface(DebugAdapterClient client, RequestHotReloadInterface requestHotReloadInterface) {
+        _sawmill.Debug("Debug adapter triggered interface hot reload");
+        try {
+            _dreamManager.HotReloadInterface();
+            requestHotReloadInterface.Respond(client);
+        } catch (Exception e) {
+            requestHotReloadInterface.RespondError(client, e.Message);
+        }
+    }
+
+    private void HandleRequestHotReloadResource(DebugAdapterClient client, RequestHotReloadResource requestHotReloadResource) {
+        _sawmill.Debug("Debug adapter triggered resource hot reload for "+requestHotReloadResource.Arguments.FilePath!);
```

Comment:
This should explicitly handle a null instead of potentially throwing.

```suggestion
        if (string.IsNullOrWhiteSpace(requestHotReloadResource.Arguments.FilePath)) {
            _sawmill.Error($"Debug adapter requested a resource hot reload but didn't provide a file");
            requestHotReloadResource.RespondError(client, "No file provided for a hot reload");
            return;
        }
        
        _sawmill.Debug("Debug adapter triggered resource hot reload for "+requestHotReloadResource.Arguments.FilePath);
```